src/Uno.UWP/Generated/3.0.0.0/Windows.UI.WindowManagement/AppWindow.cs

```diff
@@ -3,12 +3,12 @@
 #pragma warning disable 114 // new keyword hiding
 namespace Windows.UI.WindowManagement
 {
-#if false
+#if __ANDROID__ || __IOS__ || IS_UNIT_TESTS || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
```

Comment:
I must have accidentally done this...

Reply:
It's confusing because there was another AppWindow (`Microsoft.UI.Windowing.AppWindow`) that you implemented and it has no generated file, which is a sync gen bug whose fix will be a large one with tons of breaking changes.